mteb/models/e5_models.py

```diff
@@ -56,10 +160,29 @@ def encode_corpus(
 
 
 e5_mult_small = ModelMeta(
-    loader=lambda: E5Wrapper("intfloat/multilingual-e5-small"),  # type: ignore
+    loader=partial(E5Wrapper, model_name="intfloat/multilingual-e5-small"),  # type: ignore
+    name="intfloat/multilingual-e5-small",
+    languages=xlmr_languages,
+    open_source=True,
+    revision="e4ce9877abf3edfe10b0d82785e83bdcb973e22e",
+    release_date=e5_paper_release_date,
+)
+
+e5_mult_base = ModelMeta(
```

Comment:
Not sure if this has been discussed before. How would different revisions of the same model affect naming here?

Reply:
Yeah, so I am not sure how to deal with a model that has multiple loaders for different revisions. Atm. you can only have one model with the same `name`. I suppose in principle we could probably allow in the way that we load using `mteb.get_model`, but atm I don't think it will be a problem. For the variable name `e5_mult_small` it can really be anything as long as it is unique within the file.